components/automate-ui/src/app/pages/project/details/project-details.component.ts

```diff
@@ -132,6 +132,16 @@ export class ProjectDetailsComponent implements OnInit, OnDestroy {
       .subscribe(() => {
         this.store.dispatch(new GetProject({ id: this.id }));
       });
+
+    // when project becomes applied need to also mark rules applied
+    this.store.select(allProjects).pipe(
```

Comment:
is there a reason we have to check all projects in the store instead of checking the current project?

Reply:
Good observation! Changed.